### PR TITLE
Title screen now uses keyboard input

### LINKED ISSUE #81: Make title screen input by keyboard and not mouse

### PR DESCRIPTION
**Pull request by OrenjiAkira**
Fixed all related input crashes, and closes #81 
Also we now have a juicy cursor on the the title screen. Yay

### CODE REVIEW

**Comment by Kazuo256** [comment]
I think it is now safe to remove the code for the old buttons. Could you do that too?

**Comment by OrenjiAkira** [comment]
Hmmm ok, I'll see about that. I probably will have to change the signal's name, though.

**Comment by OrenjiAkira** [comment]
There, done.

File: components/controller/menu_controller.gd [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  choice = (choice % choice_list_size) + 1
+  update()
+
+func event_select():
+  var selection = saves_node.get_child(choice)
+  if saves_node.get_child(choice) != cursor:
+    if selection.get("route_id"):
+      selection.emit_signal("selected")
+    else:
+      selection.emit_signal("selected")
```
Comment (Kazuo256) [inline components/controller/menu_controller.gd]: This code part is very interesting. I think you probably don't need the `selection.get("route_id")` thing.

Comment (OrenjiAkira) [inline components/controller/menu_controller.gd]: Woops, I think I changed it and forgot to clean that up. I will fix it

Comment (OrenjiAkira) [inline components/controller/menu_controller.gd]: Also what do you mean by interesting; are you patronizing me?

File: components/controller/menu_controller.gd [hunk 2]
```diff
@@ -1,49 +1,43 @@
...
   choice = ((choice + choice_list_size - 2) % choice_list_size) + 1
-  update()
+  update_choice()
 
 func event_down():
   choice = (choice % choice_list_size) + 1
-  update()
+  update_choice()
 
 func event_select():
```
Comment (OrenjiAkira) [inline components/controller/menu_controller.gd]: There. Is it ok now?

File: components/controller/menu_controller.gd [hunk 3]
```diff
@@ -1,49 +1,43 @@
...
-    if selection.get("route_id"):
-      selection.emit_signal("selected")
-    else:
-      selection.emit_signal("selected")
-    self.disable()
-  else:
-    print("Error! This shouldn't happen! You can't select the cursor to load!")
+  assert(selection != cursor) # redundant check yay
+  selection.emit_signal("selected")
+  self.disable()
```
Comment (OrenjiAkira) [inline components/controller/menu_controller.gd]: There. Is it ok now?

File: components/controller/menu_controller.gd (hunk 1 above)
Comment (Kazuo256) [inline components/controller/menu_controller.gd]: I just wanted to say it was funny. My bad.

**Pull request merged**